src/app/lib/config/optimizely/index.js

```diff
@@ -1,10 +1,34 @@
-export default {
-  // This is the key of the whole experiment
-  flagKey: 'newswb_ws_topbarojs_read_more',
-  // This is the key for the 'rule' that is attached to the experiment
-  ruleKey: 'newswb_ws_topbarojs_read_more',
-  variationMappings: {
-    on: 'on',
-    off: 'off',
+const OPTIMIZELY_CONFIG = {
+  newswb_ws_topbarojs_read_more: {
+    // This is the key of the whole experiment
+    flagKey: 'newswb_ws_topbarojs_read_more',
+    // This is the key for the 'rule' that is attached to the experiment
+    ruleKey: 'newswb_ws_topbarojs_read_more',
```

Comment:
Are these used anymore? As it appears we're using the experiment name as the lookup? (I would happily remove any extra config that's not needed from here).

Reply:
Or (not sure if it's part of this or not), could we consider deleting one of these, as they have the same values, and renaming to `experiment` or `experimentName` - I don't believe that we need to really need to know about or match our variable names to Optimizely) 

```suggestion
    // This is the key of the whole experiment
    experimentName: 'newswb_ws_topbarojs_read_more',
```

Reply:
The work in this spike means we're not using this Config file at all anymore.

However, do you think it would be useful having some type of config to keep track of what experiments are running? E.g. keeping a list of experiment names.

Another approach we considered was keeping all the running experiments in a provider. So having a Provider instead of a Config. Though we would need to check the impact of re-renders.